a3p-integration/proposals/d:quick-send/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "agoricProposal": {
+    "source": "subdir",
+    "sdk-generate": [
+      "vats/init-orchestration.js",
+      "orchestration/init-quickSend.js"
```

Comment:
in `fast-usdc` now, yes? 

I'm reluctant to have `quickSend` around as a synonym for fast-usdc but we could use that to mean "the prototype". But what would be more clear is to call it that, like `fu-prototype`. Let's try to get the naming of things figured out before merging this, and document it in the fast-usdc README.md